apps/python/register/build.sh

```diff
@@ -0,0 +1,6 @@
+DEFAULT_IMAGE_NAME="pocket_llm_register" # vllm.entrypoints.openai.api_server --> "/v1/chat/completions" & "/v1/completions"
```

Comment:
The "register" will only work for llm tasks?
I think this should be a general module that serves registration for any dataset that we add, for any ML model.